components/languageSelector/LanguageSelect.tsx

```diff
@@ -0,0 +1,39 @@
+import { twMerge } from 'tailwind-merge';
+
+interface Option {
+  value: string;
+  text: string;
+}
+
+interface LanguageSelectProps {
+  className?: string;
+  onChange: (value: string) => void;
+  options: Option[];
```

Comment:
```suggestion
  onChange?: (value: string) => void;
  options?: Option[];
```

Reply:
I think you should use the `@types/form/Select.ts` types instead of defining again since this dropdown is basically a Select component.